utils/filters.py

```diff
@@ -27,3 +27,12 @@ def filter_ignored_users(self, msg, cmd, args, dry_run):
         if msg.frm.nick in self.bot_config.IGNORE_USERNAMES:
             return None, None, None
         return msg, cmd, args
+
+    @cmdfilter
+    def filter_private_diverts(self, msg, cmd, args, dry_run):
+        """
+        Leave a message that response is diverted as private message.
+        """
+        if cmd in self.bot_config.DIVERT_TO_PRIVATE:
+            self.send(msg.frm, 'It is diverted into private message, check DM with @co-robo')
```

Comment:
per https://github.com/coala/corobo/issues/349 , I think this message should refer to the caller, so they know it is a response to them.  (otherwise everyone goes looking for a DM.)

Reply:
okay.. I will fix that